bumble/hci.py

```diff
@@ -732,19 +727,13 @@ def phy_list_to_bits(phys: Optional[Iterable[int]]) -> int:
 
 HCI_LE_PHY_BIT_NAMES = ['LE_1M_PHY', 'LE_2M_PHY', 'LE_CODED_PHY']
 
-HCI_LE_PHY_TYPE_TO_BIT = {
-    HCI_LE_1M_PHY:    HCI_LE_1M_PHY_BIT,
-    HCI_LE_2M_PHY:    HCI_LE_2M_PHY_BIT,
-    HCI_LE_CODED_PHY: HCI_LE_CODED_PHY_BIT
+HCI_LE_PHY_TYPE_TO_BIT: dict[int, int] = {
+    Phy.LE_1M:    1,
```

Comment:
This is not the same value as before (supposed to be the bit offset for PHY bit lists), it will break other methods.

Reply:
Done.